demosplan/DemosPlanStatementBundle/Logic/DraftStatementService.php

```diff
@@ -1384,16 +1384,16 @@ protected function createSingleDocumentVersion(SingleDocument $singleDocument)
      * Convert Result to Legacy.
      *
      * @param array       $list
+     * @param string      $procedureId
      * @param string      $search
      * @param array|null  $filters
      * @param array|null  $sort
      * @param string|null $manualSortScope
```

Comment:
```suggestion
```

Reply:
See above, without defining the param types this should remain